lib/rules/no-missing-attrs.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
       },
     ],
     messages: {
-      [MESSAGE_IDS.MISSING_ATTRS]: "Some attributes {{terms}} are missing.",
+      [MESSAGE_IDS.MISSING_ATTRS]: "This element is detected as text and should have the {{terms}} pipe.",
```

Comment:
Shouldn't it be attribute (notranslate, translate) or directive (translate)?

Reply:
:smile: guys we are overcomplicating :)

Reply:
Well, it should be correct, yes it should be "directive" 👍 
(Also keep in mind that this repo incl. its PRs and discussion is public)

Reply:
so just to be clear it should be "This element is detected as text and should have the (notranslate, translate) directive." agree?

Reply:
@dersphere, @2bix ☝🏼

Reply:
Why is it public? 

And I would more likely stick with attribute

Reply:
well me too, is more generic which is the reality because I'm not validating (pipes, directives) etc, just an attribute.